Update Core Node Admin Guide (#376)
stellar/stellar-docs: network/core-node/admin-guide/soroban-settings.mdx

---
title: Soroban Settings
sidebar_position: 110
---

import { Alert } from "@site/src/components/Alert";

> Soroban has a large collection of settings stored on-ledger that can be modified through a validator vote. Headers, we will describes how you can propose a new settings upgrade, and how to examine a proposed upgrade. You can also look at the [Commands page](./commands.mdx) for more details on the stellar-core commands used below.

Reviewer: ```suggestion
Soroban has a large collection of settings stored on-ledger that can be modified through a validator vote. Here you can find out how to propose a new settings upgrade as well as how to examine a proposed upgrade. You can also look at the [Commands page](./commands.mdx) for more details on the stellar-core commands used below.
```
